addons/weapononback/functions/fnc_onDragWOB.sqf

```diff
@@ -0,0 +1,99 @@
+#include "script_component.hpp"
+/*
+ * Author: BaerMitUmlaut
+ * Handles dragging the weapon on the back onto a container.
+ *
+ * Arguments:
+ * 0: Secondary slot control <CONTROL>
+ * 1: Mouse button ID <NUMBER>
+ *
+ * Return Value:
+ * None
+ *
+ * Public: No
+ */
+params ["_ctrl", "_mouseButton"];
+
+if (_mouseButton != 0 || {secondaryWeapon ACE_player != QGVAR(weapon)}) exitWith {};
+
+// Replace primary slot with drop catcher, make background white
+private _display = ctrlParent _ctrl;
+private _primarySlot = _display displayCtrl IDC_PRIMARY_SLOT;
+_primarySlot ctrlSetFade 1;
+_primarySlot ctrlCommit 0;
+private _primaryBG = _display displayCtrl IDC_PRIMARY_BG;
+_primaryBG ctrlSetText "#(rgb,8,8,3)color(1,1,1,0.25)";
+
+private _dropCatcher = _display ctrlCreate ["ctrlActivePictureKeepAspect", IDC_DROP_CATCHER];
```

Comment:
`// RscPicture does not trigger LBDrop` same as in `fnc_onDrag` to keep it consistent and clear?